superset-frontend/plugins/legacy-preset-chart-deckgl/src/DeckGLContainer.tsx

```diff
@@ -58,6 +60,19 @@ export const DeckGLContainer = memo(
     const [lastUpdate, setLastUpdate] = useState<number | null>(null);
     const [viewState, setViewState] = useState(props.viewport);
     const prevViewport = usePrevious(props.viewport);
+    const glContextRef = useRef<WebGL2RenderingContext | null>(null);
+
+    useEffect(
+      () => () => {
+        if (glContextRef.current) {
```

Comment:
Maybe we can shorten this up a bit without the if statements

`glContextRef.current?.getExtension('WEBGL_lose_context')?.loseContext();`